### ISSUE TITLE
How to turn off auto-generation of inject code in onCreateView?

### ISSUE THREAD

**Issue by ar-g** [avast/android-butterknife-zelezny#108]
It is pretty annoying to get this code every time I wanna update binding of views because I do bind my views in my parent class.  I would like to have an option to turn it off.

'''
@Override public View onCreateView(LayoutInflater inflater, ViewGroup container,
      Bundle savedInstanceState) {
    // TODO: inflate a fragment view
    View rootView = super.onCreateView(inflater, container, savedInstanceState);
    ButterKnife.bind(this, rootView);
    return rootView;
  }
'''

**Comment by shakil807g** [comment]
yes i totally agree @ar-g  with It is pretty annoying to get this code every time  please  remove it . give some option off this code

**Comment by yuraj11** [comment]
👍

**Comment by ParfoisMeng** [comment]
@ar-g @martinvano @shakil807g 
Maybe you can try this. If it helps you, please give me a star.
https://github.com/avast/android-butterknife-zelezny/issues/164
